modules/store/testing/src/testing.ts

```diff
@@ -20,6 +21,7 @@ export interface MockStoreConfig<T> {
 export function provideMockStore<T = any>(
   config: MockStoreConfig<T> = {}
 ): Provider[] {
+  setNgrxMockEnvironment(true);
```

Comment:
Can there also be a test to verify that `setNgrxMockEnvironment` is called here?